mass_editing/__openerp__.py

```diff
@@ -20,32 +20,25 @@
 #
 ##############################################################################
 {
-    "name": "Mass Editing",
-    "version": "8.0.1.3.0",
-    "author": "Serpent Consulting Services,Odoo Community Association (OCA)",
-    "contributors": [
-        "Oihane Crucelaegui <oihanecrucelaegi@gmail.com>",
+    'name': 'Mass Editing',
+    'version': '9.0.1.0.0',
+    'author': 'Serpent Consulting Services Pvt. Ltd., '
+              'Odoo Community Association (OCA)',
+    'contributors': [
+        'Oihane Crucelaegui <oihanecrucelaegi@gmail.com>',
+        'Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>',
     ],
-    "category": "Tools",
-    "website": "http://www.serpentcs.com",
-    "license": "GPL-3 or any later version",
-    "description": """
-    This module provides the functionality to add, update or remove the values
-    of more than one records on the fly at the same time.
-    You can configure mass editing for any OpenERP model.
-    The video explaining the features and how-to for OpenERP Version 6
-    is here http://t.co/wukYMx1A
-    The video explaining the features and how-to for OpenERP Version 7 is
-    here : http://www.youtube.com/watch?v=9BH0o74A748&feature=youtu.be
-    For more details/customization/feedback contact us on
-    contact@serpentcs.com.
-    """,
+    'category': 'Tools',
+    'website': 'http://www.serpentcs.com',
+    'license': 'GPL-3 or any later version',
+    'summary': 'Mass Editing',
     'depends': ['base'],
+    'images': [],
     'data': [
-        "security/ir.model.access.csv",
+        'security/ir.model.access.csv',
         'views/mass_editing_view.xml',
     ],
-    'installable': False,
+    'installable': True,
     'application': True,
```

Comment:
Does this fit the definition of an application? Seems more like this module adds functionality to existing applications IMO